custom-words.txt

```diff
@@ -16,6 +16,7 @@ ACCELCONTAINER
 acceptors
 accesskey
 accesspoint
+accesspolicies
```

Comment:
You shouldn't need to add this. You should instead be using PascalCase, CamelCase versions of it, or just using plain english documentation "access polices"

Reply:
Still true, I think?

Reply:
(and nonbreaking if so)